master/reference/felix/configuration.md

```diff
@@ -19,75 +19,73 @@ The full list of parameters which can be set is as follows.
-| ReportingIntervalSecs                   | FELIX_REPORTINGINTERVALSECS             | 30                                   | Interval at which Felix reports its status into the datastore or 0 to disable.  Must be non-zero in OpenStack deployments.  |
-| ReportingTTLSecs                        | FELIX_REPORTINGTTLSECS                  | 90                                   | Time-to-live setting for process-wide status reports. |
-| IpInIpMtu                               | FELIX_IPINIPMTU                         | 1440                                 | The MTU to set on the tunnel device. See [Configuring MTU]({{site.baseurl}}/{{page.version}}/usage/configuration/mtu) |
+|  Setting                         | Environment variable            | Description | Schema | Default |
+|  ------------------------------- | ------------------------------- | ----------- | ------ | ------- |
+|  DatastoreDriver                 | FELIX_DATASTOREDRIVER           | The datastore that Felix should read endpoints and policy information from. | etcdv2, kubernetes | etcdv2 |
+|  FelixHostname                   | FELIX_FELIXHOSTNAME             | The hostname Felix reports to the plugin. Should be used if the hostname Felix autodetects is incorrect or does not match what the plugin will expect. | string | socket.gethostname() |
+|  LogFilePath                     | FELIX_LOGFILEPATH               | The full path to the felix log. Set to `none` to disable file logging. | string | /var/log/calico/felix.log |
+|  LogSeveritySys                  | FELIX_LOGSEVERITYSYS            | The log severity above which logs are sent to the syslog. | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive)  | INFO |
+|  LogSeverityFile                 | FELIX_LOGSEVERITYFILE           | The log severity above which logs are sent to the log file. | Same as LogSeveritySys | INFO |
+|  LogSeverityScreen               | FELIX_LOGSEVERITYSCREEN         | The log severity above which logs are sent to the stdout. | Same as LogSeveritySys | INFO |
+|  StartupCleanupDelay             | FELIX_STARTUPCLEANUPDELAY       | Delay, in seconds, before felix does its start-of-day cleanup to remove orphaned iptables chains and ipsets. Before the first cleanup, felix operates in "graceful restart" mode, during which it preserves any pre-existing chains and ipsets. In a large deployment you may want to increase this value to give felix more time to load the initial snapshot from etcd before cleaning up. | int | 30 |
```

Comment:
That was not an intentional change, good catch. Must have been an autocorrect spelling from somewhere in my work flow ...